modules/oadp-release-notes-1-1-1.adoc

```diff
@@ -0,0 +1,16 @@
+
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-1_{context}"]
+= OADP 1.1.1 release notes
+
+[id="known-issues-1-1-1_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+* OADP currently does not support backup and restore of AWS EFS volumes using restic in Velero (link:https://issues.redhat.com/browse/OADP-778[*OADP-778*]).
+
+* CSI backups might fail due to a Ceph limitation of 100 `VolumeSnapshotContent` snapshots per PVC (link:https://issues.redhat.com/browse/OADP-804[*OADP-804*]).
```

Comment:
This needs to be confirmed, unsure if this is true, please re-check.

Reply:
@shubham-pampattiwar -- This is my source -- https://issues.redhat.com/browse/OADP-804  -- does it need more confirmation? Who do I ask, if it does?

Reply:
@eemcmullan Can you please confirm this release note ?

Reply:
As https://issues.redhat.com/browse/OADP-804 has been updated, it looks like this is confirmed and detailed here: https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.11/html/managing_and_allocating_storage_resources/volume-snapshots_rhodf